Keras_Engine/Keras_Engine.csproj

```diff
@@ -116,4 +122,8 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>rem Installing keras in python at compile time
+$(SolutionDir)/../Python_Toolkit/Python_PostBuild/bin/Debug/Python_Postbuild.exe "pillow" "tensorflow==2.0"</PostBuildEvent>
```

Comment:
This wasn't working for me - but if I removed `Installing keras in python at compile time` then it did. With that in, it had a 9009 error, but removing that would make it build (but Python did not install - more on that in Python_Toolkit).